tests/benchmark/test_benchmark.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np 
+import shap
+
+model = lambda x: np.array([np.linalg.norm(x)])
+X = np.array([[3, 4], [5, 12], [7, 24]])
+y = np.array([5, 13, 25])
+explainer = np.array([[-1, 2], [-4, 2], [1, 2]])
+masker = X
+
+def test_update():
```

Comment:
great to see some tests!

Reply:
are these configured to run in the build already?  these aren't actually running the benchmarking from what I'm reading, it's just a few small tests of the benchmark class (update/get_benchmark/get_metrics), so execution time isn't a concern.

Reply:
Will include tests for build in the next PR along with tokenizer support tests